### ISSUE TITLE
ambassador flamelash script

### ISSUE THREAD

**Issue by hraesvelgr** [ClassicWoW/Nefarian_1.12.1_Bugtracker#1067]
das script für flamelash funktioniert momentan nur halb.

ist:
während des kampfes spawnt ein(1) elemtar und hängt öde an seinem spawnpunkt rum. 

soll: 
es sollten nach und nach mehrere elementare an allen 4 runen im raum spawnen, die, wenn sie den boss erreichen, sterben und ihm einen buff geben. die elementare sollten auf den boss zuwandern, jedoch beim ersten schaden den sie bekommen aggro ziehen und die spieler angreifen.

- By adashiel (271,701 – 12·74·921) on 2008/05/12 (Patch 2.4.1)		
Throughout the fight Burning Spirit elementals spawn and head toward him. If they reach him they will die and give him Burning Spirit, a buff that increases his damage, and stacks up to 50 times. However, if you aggro the elementals before they reach him, they won't buff him, and you can AoE them or otherwise take them out.

flamelash: http://datenbank.welli-it.de/?npc=9156
elementare: http://datenbank.welli-it.de/?npc=9178
buff auf flamelash: http://datenbank.welli-it.de/?spell=14744
buff auf die elementare, wenn sie flamelash erreichen: http://datenbank.welli-it.de/?spell=13489

**Comment by Gnomorus** [comment]
Bin gespannt, ob der Kommentar zu dem BC Patchstand als Beweis durchgeht.
Sicher ist auf jeden Fall, das der Bosskampf ein kleines Event war, was er hier bei weitem nicht ist.

**Comment by hraesvelgr** [comment]
dann find mal kommentare dazu, die von classic sind. zumal: was wurd ein bc an brd geändert? wirklich viele änderungen an den kleinen instanzen gabs in bc nicht.

**Comment by Gnomorus** [comment]
https://www.youtube.com/watch?v=v8shrjU6LvU (classic)
https://www.youtube.com/watch?v=T7xnH8lKPbw (BC)

**Comment by hraesvelgr** [comment]
ich kontne da ehrlich gesagt nahezu nix erkennen, ausser genau einen spawn auf allen runen. 

in beiden videos-

**Comment by Seraphitas** [comment]
Das Classic Video zeigt auf jeden Fall eindeutig das viele Elementare spawnen. Ich hab selbst vor 10 Jahren Classic gespielt und weiß das da viele von den Eles waren

**Comment by hraesvelgr** [comment]
ja, eben auf jeder rune im raum 1 elemtentar pro welle. aber die frequenz der wellen ist das, was ich wissen wollte
